lib/src/firefuel_collection.dart

```diff
@@ -8,22 +8,43 @@ import 'package:firefuel/src/collection.dart';
 abstract class FirefuelCollection<T extends Serializable>
     implements Collection<T> {
   final String collectionName;
+  final FirebaseFirestore firestoreInstance;
 
-  FirefuelCollection(this.collectionName);
+  const FirefuelCollection(
+    this.collectionName, {
+    required this.firestoreInstance,
+  });
+
+  /// Converts a [DocumentSnapshot] to a [T]
+  T fromFirestore(
+    DocumentSnapshot<Map<String, dynamic>> document,
+    SnapshotOptions? options,
+  );
+
+  /// Converts a [T] to a [Map<String, dynamic>] to upload to Firestore.
+  Map<String, Object?> toFirestore(
```

Comment:
The type you've added to the documentation and the type you require in the method signature are different. I think you should update your doc comment appropriately.

Reply:
Good catch! Thanks 👍